src/mds/MDCache.cc

```diff
@@ -7912,7 +7912,9 @@ void MDCache::shutdown_start()
 bool MDCache::shutdown_pass()
 {
   dout(5) << "shutdown_pass" << dendl;
-  ceph_assert(kill_shutdown_at != KILL_SHUTDOWN_AT::SHUTDOWN_START);
+  //ceph_assert(kill_shutdown_at != KILL_SHUTDOWN_AT::SHUTDOWN_START);
+  if (kill_shutdown_at == KILL_SHUTDOWN_AT::SHUTDOWN_START)
+    _exit(134);
```

Comment:
134 is not a valid exit code. I'm using `120`:

https://github.com/ceph/ceph/pull/56997/files#diff-56ee22e891f482afe8d7ea22e9ad4f49d15698e2546c59187fbce6d34474ab96R1270

Let's just be consistent with that.